docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -37,7 +37,11 @@ return [
 ];
 ```
 
-The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration.
+The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration. 
+
+WARNING: **Cache Named `config` Is Not Possible**
+A cache named `config` is not possible due to internal service conflicts with MVC configuration.
+The service named `config` is reserved for project configuration and thus cannot be used with the `caches` configuration.
```

Comment:
Cache name must be unique identifier in container. Not just config but any conflicting name is not possible.

Reply:
yah but thats usually up to the user while it is not for `config` as we now do have explicit check within the abstract static factory.
So I think this might be good enough for now. We still should tackle this specific issue within `laminas-servicemanager`.